src/esp/sensor/Sensor.h

```diff
@@ -131,6 +132,13 @@ class Sensor : public Magnum::SceneGraph::AbstractFeature3D {
 class SensorSuite {
  public:
   void add(const Sensor::ptr& sensor);
+
+  /**
+   * @brief Concatenate sensorSuite's sensors to existing sensors_
+   * @param[in] sensorSuite Instance of SensorSuite class from which to copy
+   * Sensors
```

Comment:
Hmm, does this follow Python's dict concatenation convention? I think in Python if you update two dicts with one another it will overwrite by default. Something to consider.

Reply:
@Skylion007 :
Interesting. No, we follow C++ convention (for example, in C++17, we can directly use [merge()](https://en.cppreference.com/w/cpp/container/map/merge), and it behaves as what we describe here.). If we prefer the other way used in python, we can change accordingly though.